book/src/SUMMARY.md

```diff
@@ -34,6 +35,7 @@
 - [Proposed Architectural Changes](proposals.md)
   - [Ledger Replication](ledger-replication-to-implement.md)
   - [Secure Vote Signing](vote-signer.md)
```

Comment:
Should we delete this line?

Reply:
Totally. Bad rebase :-/
Thanks for catching it!